nova/scheduler/utils.py

```diff
@@ -917,8 +917,21 @@ def request_is_resize(spec_obj):
         return False
     if 'scheduler_hints' not in spec_obj:
         return False
-    check_type = spec_obj.scheduler_hints.get('_nova_check_type')
-    return check_type == ['resize']
+    check_type = spec_obj.get_scheduler_hint('_nova_check_type')
+    return check_type == 'resize'
```

Comment:
Just as explanation: This used to be a list, because `request_is_rebuild()` is upstream code and we modeled this function after it. With you changing upstream code, that's not necessary anymore for sure :)